docs/tutorials/bitcoin-price-feed/contract.md

```diff
@@ -56,11 +50,11 @@ contract PriceFeed is UsingWitnet {
 }
 ```
 
-What you are doing here is quite straightforward:
+Executing the above will:
 
 - Import `UsingWitnet.sol` so your contract is Witnet-enabled.
 - Import `BitcoinPrice.sol` so that you can instantiate the Witnet
-  request when needed.
+  request when necessary.
 - Make your contract inherit `UsingWitnet`.
```

Comment:
```suggestion
- Make your contract inherits `UsingWitnet`.
```

Reply:
This is correct - similar grammar to "Let your family inherit"